.gitignore

```diff
@@ -15,6 +15,7 @@
 *.7z
 *.dmg
 *.gz
+!*.nii.gz
```

Comment:
This line can be removed as you have explicitly added the test file.

Reply:
Done